EmulatorDetectorProject/EmulatorDetector/src/main/java/net/skoumal/emulatordetector/EmulatorDetector.java

```diff
@@ -26,56 +29,126 @@
      * Detects if app is currenly running on emulator, or real device.
      * @return true for emulator, false for real devices
      */
-    public static boolean isEmulator() {

```

Comment:
Please do not rename this method, it should stay as is for backward compatibility.